From 827da5aff80ce6e87d9efda5433f4cf54d6a4723 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 7 Mar 2019 04:13:25 -0800
Subject: [PATCH 0551/1852] dt-bindings: media: xilinx: csi2rxss: Add RAW16 and
 RAW20 pixel format strings

commit 453ec72356b1f7a8bfdc083a95a31715c0895187 from
https://github.com/Xilinx/linux-xlnx.git

Adds RAW16 and RAW20 as valid string inputs for xlnx,csi-pxl-format
device tree property.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
index 453dcac8b22b..20460744b4c6 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
@@ -37,7 +37,7 @@ Required properties:
 - xlnx,csi-pxl-format: This denotes the CSI Data type selected in hw design.
   Packets other than this data type (except for RAW8 and User defined data
   types) will be filtered out. Possible values are RAW6, RAW7, RAW8, RAW10,
-  RAW12, RAW14, RGB444, RGB555, RGB565, RGB666, RGB888 and YUV4228bit.
+  RAW12, RAW14, RAW16, RAW20, RGB444, RGB555, RGB565, RGB666, RGB888 and YUV4228bit.
 
 - xlnx,vfb: Video Format Bridge, Denotes if Video Format Bridge is selected
   so that output is as per AXI stream documented in UG934.
-- 
2.31.1

